### PR TITLE
Set default partition info in partition_gen.sh

### PR DESCRIPTION
**Pull request by junmin-kim**
If CONFIG_ARTIK053_FLASH_PART_LIST or CONFIG_ARTIK053_FLASH_PART_NAME are unset, they will be set as default value

### CODE REVIEW

File: build/configs/artik053/tools/openocd/partition_gen.sh [hunk 1]
```diff
@@ -36,8 +36,8 @@ OPENOCD_DIR_PATH=${BOARD_DIR_PATH}/tools/openocd
 FLASH_BASE=0x04000000
 
 # Partition information
-partsize_list=${CONFIG_ARTIK053_FLASH_PART_LIST}
-partname_list=${CONFIG_ARTIK053_FLASH_PART_NAME}
+partsize_list=${CONFIG_ARTIK053_FLASH_PART_LIST:="16,48,192,32,512,2400,1536,1536,1400,8,512,"}
```
Comment (sunghan-chang) [inline build/configs/artik053/tools/openocd/partition_gen.sh]: No. I can't do this. If default values are changed, we should modify here together. That is too unconvience. We should get them from Kconfig.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: build/configs/artik053/tools/openocd/partition_gen.sh [hunk 2]
```diff
@@ -31,13 +31,17 @@ OS_DIR_PATH=${PWD}
 BUILD_DIR_PATH=${OS_DIR_PATH}/../build
 BOARD_DIR_PATH=${BUILD_DIR_PATH}/configs/${BOARD_NAME}
 OPENOCD_DIR_PATH=${BOARD_DIR_PATH}/tools/openocd
+ARTIK053_KCONFIG=${OS_DIR_PATH}/arch/arm/src/artik053/Kconfig
```
Comment (manoharahk) [inline build/configs/artik053/tools/openocd/partition_gen.sh]: artik053 -> ${BOARD_NAME}

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**